src/netbase.cpp

```diff
@@ -358,14 +359,17 @@ bool Socks5(const std::string& strDest, uint16_t port, const ProxyCredentials* a
             return false;
```

Comment:
nit - Noticed this when reviewing the changes. Since we're already changing the file - should Line 357/358 be a `LogError` not a `LogPrintf`?